packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js

```diff
@@ -76,11 +78,19 @@ function VideoIntelligenceServiceClient(gaxGrpc, loadedProtos, opts) {
     grpc: gaxGrpc.grpc
   }).operationsClient(opts);
 
+  var protoFilesRoot = new gax.grpc.GoogleProtoFilesRoot();
+  protoFilesRoot = protobuf.loadSync(
```

Comment:
Is this right? The line before this assigns a value to `protoFileRoot`, then this overwrites it?

Reply:
Yep the root gets passed into the loader and uses that as a base for the protos that are loaded. The method _probably_ updates the passed in object but I'd rather not rely on that assumption.

Reply:
Ah, I missed that, sorry!